src/app/components/ConsentBanner/index.stories.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import {
+  ConsentBanner,
+  ConsentBannerButton,
+  ConsentBannerLink,
+  ConsentBannerText,
+} from '.';
+
+const Accept = <ConsentBannerButton onClick={() => {}}>OK</ConsentBannerButton>;
+
+const Reject = (
+  <ConsentBannerLink href="https://hello.com" onClick={() => {}}>
```

Comment:
```suggestion
  <ConsentBannerLink href="https://www.bbc.co.uk/usingthebbc/your-data-matters">
```

What's the `onClick` for? Don't think we should have this for links (`Find out more`), only buttons (`OK`)